doc/regression-tests.md

```diff
@@ -16,79 +112,83 @@ Note that the `RUN_REGRESSION_TESTS` environment variable can also be used in co
 
-## How to get the regression tests to pass after a code change
+* [PR #2161](https://github.com/alphagov/smart-answers/pull/2161) e.g. [Regenerate test artefacts for am-i-getting-minimum-wage](https://github.com/alphagov/smart-answers/commit/f92e4e0033bbb1915f57e33cf749834c0d843987)
+* [PR #2181](https://github.com/alphagov/smart-answers/pull/2181) e.g. [Update regression test artefacts for question pages](https://github.com/alphagov/smart-answers/commit/023a64c4c288194d4a5f94a68e824cea8b272816)
 
-This list guides you through the steps you need to take to make the smallest change to a smart-answer, for example changing the text on a landing page.
+### Unix tree view
 
-1. Make the change to the code/template
-2. Commit the change
-3. Run the [regression tests](#regression-tests)
-4. Commit changes to the [test artefacts](#artefacts)
-5. Re-run the regression tests
-6. Regenerate the [checksums](#checksums)
-7. (Optional) Re-run the regression tests
-8. Commit the changes to the checksums
+The Unix `tree` command can be used to generate an ASCII-art view of the relevant artefacts directory or sub-directory. You can generate this view before and after a change and include them in the commit note with annotations explaining what has changed and why. Here is an example:
 
-Once the regression tests have passed you can prepare your changes for review as usual:
+* [PR #2529](https://github.com/alphagov/smart-answers/pull/2529) e.g. [Update regression test responses, expected results & artefacts](https://github.com/alphagov/smart-answers/commit/b225746fd044c5699075778fc1b507662c881df5)
 
-1. Create a [pull request](pull-requests.md)
-2. (Optional) Deploy an app to [Heroku](factcheck.md#deploying-to-heroku) for factcheck
-  1. Add a "Waiting for factcheck" label to the pull request
+## Developer workflow
 
-## Continuous integration
+It should be obvious from the [Structuring commits](#structuring-commits) & [Artefact changes](#artefact-changes) sections above that there isn't one "correct" way to organise your commits in a pull request. However, as a starting point, here are the steps you might use to implement a simple content change:
 
-The [main CI instance](doc/continuous-integration.md#main) and the [corresponding branches one](doc/continuous-integration.md#branches) run the rake `test` task and so work in the same way as above.
+1. Make the change to the relevant template file
+2. [Run the regression tests](#running-regression-tests) thus re-generating the artefacts
+3. Review the changes to the [artefacts](#artefact-files) and check they are as you expect
+4. If artefact changes are as expected, [re-generate the checksums](#checksum-file)
```

Comment:
Typo: re-generate should be spelt regenerate (no hypen). See http://dictionary.cambridge.org/dictionary/english/regenerate

[Regeneration](https://en.wikipedia.org/wiki/Regeneration_(Doctor_Who)) has been a common concept since 1966, so seeing it misspelt is even more odd.